class Solution:
    def two_sum(self, nums, target):
        raise ValueError("No two sum solution found")

